app/buck2_validation/src/cached_validation_result.rs

```diff
@@ -43,7 +43,7 @@ impl ValidationFailedUserFacingError {
     pub(crate) fn rendered_message(&self) -> Cow<str> {
         self.short_message.as_deref().map_or_else(
             || Cow::Borrowed("Diagnostic message is missing from validation result"),
-            |x| Cow::Owned(format!("\"{}\"", x)),
+            |x| Cow::Owned(format!("{}", x)),
         )
     }
```

Comment:
With this change it could be just simplified to return `&str`:

```
pub(crate) fn rendered_message(&self) -> &str {
    self.short_message.as_deref().unwrap_or("Diagnostic message is missing from validation result")
}
```